src/paradigma/gait_analysis.py

```diff
@@ -120,9 +122,9 @@ def extract_arm_swing_features(input_path: str, output_path: str, config: ArmSwi
     # perform principal component analysis on the gyroscope signals to obtain the angular velocity in the
     # direction of the swing of the arm 
     df[config.velocity_colname] = pca_transform_gyroscope(
-        df=df, 
-        y_gyro_colname=DataColumns.GYROSCOPE_Y,
-        z_gyro_colname=DataColumns.GYROSCOPE_Z,
+        df=df,
+        y_gyro_colname=DataColumns.GYROSCOPE_Y.value,
```

Comment:
Looking at the other examples however I can see that setting method inputs to the `DataColumns` class would not be sufficient.

Examples such as: `arr_ppg = df_ppg[DataColumns.PPG.value].to_numpy()` would still require `.value`. 

I will look at the rest of the PR before I make any further suggestions.

Reply:
I'm actually doubting now whether the Enums are a good idea. They seemed nice for explicit typing, but they will also limit end-users from using their own custom strings...

Reply:
That's a good point. Also, the Enums don't allow much flexibility that I was hoping for. For example it is not simple to make a class `GaitColumns(DataColumn)` which extends `DataColumn(Enum)`, due to some `Enum` specific limitations.

Reply:
I tried using 

```
def __str__(self):
        return self.value
    
def __repr__(self):
        return self.value
```

to avoid the need for `.value`, however, I am getting errors when working with pd dataframes. I am getting `KeyError`. I even tried using `StrEnum` hoping that it would work better but it doesn't. I will remove `Enums`